app/src/main/java/org/mozilla/tv/firefox/pocket/PocketFeedStateMachine.kt

```diff
@@ -21,9 +22,9 @@ import org.mozilla.tv.firefox.pocket.PocketVideoRepo.FeedState.FetchFailed
  *
  * ### Diagram
  * [Loading] ------------------
- *    |  ^                    |     ---
- *    V  |                    V     | V
- * [FetchFailed] ---------> [LoadComplete]
+ *    |  ^                    |           ---
+ *    V  |                    V           | V
+ * [FetchFailed] ---------> [LoadComplete] / [Inactive]
```

Comment:
Actually `Inactive` goes to `Loading` and could end up in `FetchFailed` if it's never been loaded before, not necessarily directly to `LoadComplete`.

Reply:
Although would be possible for the repo to attempt to load, the state machine will never emit anything new if its cached state is `Inactive`.

Reply:
To correct myself, as currently implemented it can only go from `Inactive` to `Inactive` or `LoadComplete`.  This should probably change though, I'm going to put some thought into it.